dotnet/samples/GettingStartedWithAgents/BedrockAgent/Step07_BedrockAgent_Declarative.cs

```diff
@@ -39,6 +39,24 @@ public async Task BedrockAgentWithConfiguration()
         await InvokeAgentAsync(agent!, "Cats and Dogs");
     }
 
+    /// <summary>
+    /// Demonstrates loading an existing Bedrock Agent.
+    /// </summary>
+    [Fact]
+    public async Task BedrockAgentWithId()
+    {
+        var text =
+            """
+            id: ${BedrockAgent:AgentId}
+            type: bedrock_agent
+            """;
+        BedrockAgentFactory factory = new();
+
+        var agent = await factory.CreateAgentFromYamlAsync(text, configuration: TestConfiguration.ConfigurationRoot);
+
+        await InvokeAgentAsync(agent!, "What is Semantic Kernel?", false);
```

Comment:
nit: For clarity I would use named parameters in this `false` here:
```suggestion
        await InvokeAgentAsync(agent!, "What is Semantic Kernel?", what: false);
```